apps/enc.c

```diff
@@ -418,15 +426,31 @@ int enc_main(int argc, char **argv)
                     BIO_printf(bio_err, "bad magic number\n");
                     goto end;
```

Comment:
I have other arguments about the 'magic' that I am keeping on my side for later discussion and enhancement, because they implied much more troubles.